Velaptor/Graphics/Renderers/ITextureRenderer.cs

```diff
@@ -96,8 +97,100 @@ public interface ITextureRenderer
     ///         <item>Texture 5</item>
     ///     </list>
     /// </remarks>
+    [SuppressMessage("ReSharper", "UnusedMemberInSuper.Global", Justification = "Public API for users.")]
     void Render(ITexture texture, int x, int y, float angle, int layer = 0);
 
+    /// <summary>
+    /// Renders the given texture at the given <paramref name="x"/> and <paramref name="y"/> coordinates <paramref name="angle"/>,
```

Comment:
Add a comma after the word `coordinates`

```suggestion
    /// Renders the given texture at the given <paramref name="x"/> and <paramref name="y"/> coordinates, <paramref name="angle"/>,
```